typescript/cli/src/commands/options.ts

```diff
@@ -150,6 +155,12 @@ export const chainCommandOption: Options = {
   description: 'The specific chain to perform operations with.',
 };
 
+export const validatorCommandOption: Options = {
+  type: 'string',
+  description: 'Comma separated list of validator addresses',
```

Comment:
Right now when we pass an argument to `builder`, we use this type `{ [key: string]: Options }`. We will likely run into type issues if we go with this option as we use our own type for commands

```
export type CommandModuleWithWriteContext<Args> = CommandModule<
  {},
  Args & { context: WriteCommandContext }
>;
```

We use this type because we want to access this `context` in the `handler` callback. I do agree that this option does give us some nice features. Happy to explore whether we can make this work as a follow up

Reply:
happy to leave this to future potential work / maybe kick the can to the product team as they've also chosen to go with comma separation